stdlib/2and3/socket.pyi

```diff
@@ -713,11 +713,11 @@ class socket:
             self, __buffers: Iterable[bytes], __ancdata: Iterable[_CMSG] = ..., __flags: int = ..., __address: _Address = ...
         ) -> int: ...
     if sys.platform == "linux" and sys.version_info >= (3, 6):
-        # TODO add the parameter types for sendmsg_afalg
-        def sendmsg_afalg(self, msg=..., *, op, iv=..., assoclen=..., flags=...) -> int: ...
+        def sendmsg_afalg(
+            self, msg: Sequence[bytes] = ..., *, op: int, iv: Any = ..., assoclen: int = ..., flags: int = ...
```

Comment:
Is `Sequence[bytes]` really the correct type? I am not familiar with this function, but this looks suspicious to me.

Reply:
From looking at the C code it seems like it should be `Iterable[bytes]`.

Reply:
I made the recommended change.